backend/src/client/MinerClient.ts

```diff
@@ -32,13 +32,13 @@ export class MinerClientImpl implements IMinerClient {
   }
 
   async power (address: string): Promise<MiningPower> {
-    const [power] = await this.callAPI<[string]>('power', [address]);
+    const [power] = await this.callAPI<[{'Power': string, 'Total': string}]>('power', [address]);
     logger.info('got power response', {
       power,
       address,
     });
 
-    if (typeof power === 'object') {
+    if (!power.Power || !power.Total) {
```

Comment:
Is there a better way to handle this check in typescript?

Reply:
Not sure, but a type guard for 'in' may have been added last year https://github.com/Microsoft/TypeScript/wiki/Roadmap#27-january-2018